pydantic/_internal/_model_construction.py

```diff
@@ -76,6 +76,7 @@ def __new__(
         namespace: dict[str, Any],
         __pydantic_generic_metadata__: PydanticGenericMetadata | None = None,
         __pydantic_reset_parent_namespace__: bool = True,
+        cls_module: str | None = None,
```

Comment:
Are you sure it's valid to pass `cls_module` as a kwarg to `__new__`?

Reply:
if module has to be passed this way for `create_model`, maybe better to use `_cls_module` to avoid clashes with config?

Reply:
Yes, it's there for `create_model`. I renamed all of them to `_cls_module`